newdle/client/src/selectors.js

```diff
@@ -43,6 +43,8 @@ export const getFullTimeslots = state =>
 export const getCreatedNewdle = state => state.creation.createdNewdle;
 
 // newdle
+export const getNewdle = state => state.newdle;
+export const getNewdleTitle = state => (state.newdle && state.newdle.title) || null;
```

Comment:
Maybe we could combine those functions that use `state.newdle` with `createSelector`? And handle cases in which `state.newdle === null` there